File: array/a5.py
# Topic: Move all the negative elements to one side of the array 
from typing import List, Any

class Array:

    def __init__(self, elements: List[Any]) -> None:
        self.elements = elements

    def sort(self) -> List[Any]:
        print(self.elements)
        return sorted(self.elements)


__array = Array([-12, 11, -13, -5, 6, -7, 5, -3, -6])
print("Move all the negative elements to one side of the array: ", __array.sort())

File: array/a5.rs
// Topic: Move all the negative elements to one side of the array 

#[derive(PartialEq, PartialOrd)]
struct Array<T> { elements: Vec<T> }

impl<T: Ord> Array<T> {
    fn sort(&mut self) {
        self.elements.sort();
    }
}

fn main() {
    let mut array  = Array { elements: vec![-12, 9, 324, -3412, 4] };
    array.sort();
    println!("Move all the negative elements to one side of the array: {:?}", array.elements);
}
